autoscheduler/scheduler/management/commands/scheduling.py

```diff
@@ -0,0 +1,148 @@
+from functools import reduce
+from itertools import islice, groupby, product
+from operator import mul
+from random import choice
+from typing import Dict, List, Tuple
+from django.core.management import base
+from scraper.models import Meeting, Section
+
+def _get_sections(course: Tuple[str, str], term: str) -> Dict[str, List[Dict]]:
+    """ Gets all sections for a given course, and organizes them by section_num
+
+    Args:
+        course: Tuple of (subject, course_num) to find sections for
+        term: Term to find sections for
+
+    Returns:
+        A dict of sections for the course with the meeting number as the key
+        and attributes for each meeting as the value
+    """
+    subject, course_num = course
+    # Create list of section_nums matching desired course
+    sections = (Section.objects.filter(course_num=course_num, subject=subject,
+                                       term_code=term)
+                .values('id', 'section_num'))
+    # Dictionary for translating meeting ids into section numbers
+    meeting_nums = {section['id']: section['section_num'] for section in sections}
```

Comment:
```suggestion
    section_nums = {section['id']: section['section_num'] for section in sections}
```
Why did I call this meeting nums when it's section numbers? Who knows but I'm not going to make a commit just to fix a variable name